cmd/tetragon/main.go

```diff
@@ -382,6 +409,31 @@ func tetragonExecute() error {
 	return obs.Start(ctx)
 }
 
+func addTracingPolicy(ctx context.Context, file string) error {
```

Comment:
since you are introducing a new helper, please make sure it's used for both files and directories.

Reply:
will do thank you